### PR TITLE
Add BIRU (Handle Service) to community showcase

### PR DESCRIPTION
**Pull request by vheuel**
BIRU is a free and unique handle service for bluesky which is managed by the Indonesian Bluesky community,  users can create their handles for free at www.bsky.makeup 

 
**list of domains** 
bsky.makeup
bsky.today
skeets.top
fess.fun
tired.baby
mlbb.skin

and still others will be added

### CODE REVIEW

**Comment by vheuel** [comment]
BIRU is a service for creating unique and free handles for Bluesky, with a choice of domains :

bsky.makeup
bsky.today
skeets.top
fess.fun

and still others will be added

File: src/data/users.tsx [hunk 1]
```diff
@@ -18,6 +18,14 @@ export type TagType =
   | 'othertools';
 
 const Users: User[] = [
+{
+    title: 'BIRU',
+    description: Free unique handle for Bluesky',
```
Comment (emilyliu7321) [inline src/data/users.tsx:23]:
```suggestion
    description: 'Assigns unique handles from various domains to Bluesky users, managed by the Indonesian community',
```

**Review by emilyliu7321 — APPROVED** [review]

**Review by emilyliu7321 — APPROVED** [review]

**Comment by Suburbanno** [comment]
Service is offline.

**Comment by gldraphael** [comment]
This service no longer works.

[www.bsky.makeup](http://www.bsky.makeup/) redirects to https://tired.baby/ with a "Domain for sale" notice.

image

**Pull request merged**